component-samples/demo/owner/README.md

```diff
@@ -162,7 +166,7 @@ Following is the list of REST response error codes and it's possible causes :
 
 # Troubleshooting
 
-As the H2 DB grows, larger heap space will be required by the application to run the service. The default configured heap size is `256 MB`. Increase the heap size appropriately in `demo/owner/owner-entrypoint.sh` to avoid heap size issue
+Increase the heap size appropriately to avoid heap size issue.
```

Comment:
Can we bring back "in demo\...sh"
Also can we say increase the heap size appropriately in case you encounter heap size issues?